docs/tools/vdb_table/data/usearch.json

```diff
@@ -53,29 +53,29 @@
     "comment": "using the multi param in index creation"
   },
   "sparse_vectors": {
-    "support": "",
+    "support": "partial",
     "source_url": "",
     "comment": ""
   },
   "bm25": {
-    "support": "",
+    "support": "none",
     "source_url": "",
     "comment": ""
   },
   "full_text": {
-    "support": "",
+    "support": "partial",
```

Comment:
reference needed here

Reply:
[Code](https://github.com/unum-cloud/usearch/blob/6b43d1ce39e0381016e121981615657af86e9568/include/usearch/index.hpp#L1608-L1610).

Reply:
The reference you provided is not clear to me. Does USearch maintain an inverted index of terms that can be queried? This is the definition for this column: https://github.com/superlinked/VectorHub/blob/7b6ab3883d400a12169b0942f21de4d51eb6eb82/docs/tools/vdb_table/vendor.schema.json#L21